### PR TITLE
Create Announcements.md

### PR DESCRIPTION
**Pull request by vemv**
Hi @seancorfield !

Here's a draft for what seem reasonable guidelines to me.

Feel free to add inline suggestions - I'd incorporate them right away.

Cheers - V

### CODE REVIEW

File: Announcements.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+## Rules and guidelines for the `#announcements` channel
+
+`#announcements` is one Clojurians' most popular channels, and a particularly benefitial one for having a healthy, cohesive community of makers.
```
Comment (eggsyntax) [inline Announcements.md]: benefitial -> beneficial

**Comment by eggsyntax** [comment]
I'm largely in agreement with the spirit here, but given the discussion in #community-development,  Sean's suggestion there to redirect people who post about blog articles to #news-and-articles, and the tweaked channel description in #announcements, I suggest putting this PR on hold for a while so we can see how things go.

If it doesn't go that way, and this is going to get merged, a few thoughts:
- I think it's worth mentioning that blog posts should go in #news-and-articles (unless we want to create a separate channel for them).
- It's worth noting that many people get notifications for posts in #announcements, which is another reason to keep it relatively low-traffic.
- The paragraph starting "Specifically" should be struck IMO; it's liable to come across as accusatory (and some folks may feel attacked by this even if it's worded pretty gently).

**Comment by vemv** [comment]
Hi @eggsyntax - thanks for the thoughts!

I have applied the suggestions.

It can be OK to keep the PR open, maybe the downside being that from time from time, people would do the precise things we seek to curb. That'd make us hesitate - should we try enforcing WIP rules or not?

Another option is further trimming this PR, and having it merged. Rules can always be strengthened later

**Comment by seancorfield** [comment]
I would not want to encourage people to use `/feed` as they're likely to subscribe a whole bunch of RSS feeds into **channels** -- and then _everyone will see those_, not just the person subscribing. That would make many channels very noisy (and would be awful in the #announcements channel).

I also don't think it is the place of Clojurians Slack to try to police how folks manage the contents of their own projects, such as readmes etc, so I'm not very comfortable with that section of the page.

**Comment by vemv** [comment]
> I would not want to encourage people to use /feed as they're likely to subscribe a whole bunch of RSS feeds into channels -- and then everyone will see those, not just the person subscribing. 

What I had meant is subscribing in a channel you own - that can be your personal Slack channel (e.g. `@vemv` in Clojurians - only I get to see it), or some channel in a different Slack workplace (e.g. #clojure in MyAwesomeCompany). Can clarify.

> I also don't think it is the place of Clojurians Slack to try to police how folks manage the contents of their own projects, such as readmes etc, so I'm not very comfortable with that section of the page.

Definitely the intent was not to police in any way how people manage their projects. It would only be a suggestion, and hopefully a useful one because probably the vast majority of folks are unaware of RSS integration to begin with (this includes myself a few days ago!).

Given Clojurians Slack is a community (as opposed to merely a medium), it has the power to improve itself. With a gentle touch, we can get people to be _more_ engaged with Clojure libs, while minimizing undesired updates.

Can refine or remove.

**Pull request closed without merging**